helpers/getLocalizeResources.js

```diff
@@ -1,408 +1 @@
-		return;
-	}
-
-	documentLocaleSettings.oslo.version = version;
-}
-
-function getVersion() {
-
-	if (documentLocaleSettings === undefined) {
-		documentLocaleSettings = getDocumentLocaleSettings();
-	}
-
-	const shouldReturnVersion =
-		documentLocaleSettings.oslo &&
-		documentLocaleSettings.oslo.version;
-	if (!shouldReturnVersion) {
-		return null;
-	}
-
-	return documentLocaleSettings.oslo.version;
-}
-
-async function shouldFetchOverrides() {
-
-	const isOsloAvailable =
-		await shouldUseBatchFetch() ||
-		shouldUseCollectionFetch();
-
-	return isOsloAvailable;
-}
-
-async function fetchOverride(formatFunc) {
-
-	let resource, res, requestURL;
-
-	if (await shouldUseBatchFetch()) {
-
-		// If batching is available, pool requests together.
-
-		resource = formatFunc();
-		res = fetchWithPooling(resource);
-
-	} else /* shouldUseCollectionFetch() == true */ {
-
-		// Otherwise, fetch it directly and let the LMS manage the cache.
-
-		resource = formatFunc();
-		requestURL = formatOsloRequest(documentLocaleSettings.oslo.collection, resource);
-
-		res = fetchCollection(requestURL);
-
-	}
-	res = res.catch(coalesceToNull);
-	return res;
-}
-
-function coalesceToNull() {
-
-	return null;
-}
-
-function formatOsloRequest(baseUrl, resource) {
-	return `${baseUrl}/${resource}`;
-}
-
-export function __clearWindowCache() {
-
-	// Used to reset state for tests.
-
-	blobs.clear();
-	cache = undefined;
-	cachePromise = undefined;
-}
-
-export function __enableDebugging() {
-
-	// Used to enable debug logging during development.
-
-	debug = true;
-}
-
-export async function getLocalizeOverrideResources(
-	langCode,
-	translations,
-	formatFunc
-) {
-	const promises = [];
-
-	promises.push(translations);
-
-	if (await shouldFetchOverrides()) {
-		const overrides = await fetchOverride(formatFunc);
-		promises.push(overrides);
-	}
-
-	const results = await Promise.all(promises);
-
-	return {
-		language: langCode,
-		resources: Object.assign({}, ...results)
-	};
-}
-
-export async function getLocalizeResources(
-	possibleLanguages,
-	filterFunc,
-	formatFunc,
-	fetchFunc
-) {
-
-	const promises = [];
-	let supportedLanguage;
-
-	if (await shouldFetchOverrides()) {
-
-		const overrides = await fetchOverride(formatFunc, fetchFunc);
-		promises.push(overrides);
-	}
-
-	for (const language of possibleLanguages) {
-
-		if (filterFunc(language)) {
-
-			if (supportedLanguage === undefined) {
-				supportedLanguage = language;
-			}
-
-			const translations = fetchFunc(formatFunc(language));
-			promises.push(translations);
-
-			break;
-		}
-	}
-
-	const results = await Promise.all(promises);
-
-	// We're fetching in best -> worst, so we'll assign worst -> best, so the
-	// best overwrite everything else.
-
-	results.reverse();
-
-	return {
-		language: supportedLanguage,
-		resources: Object.assign({}, ...results)
-	};
-}
+export { getLocalizeOverrideResources, getLocalizeResources } from '@brightspace-ui/intl/helpers/getLocalizeResources.js';
```

Comment:
So is the plan that we'll clean up places using the core alias and then fully remove this, or are there too many?

Reply:
I think it makes sense that if you are using these with the mixin you'd use them from core, but ultimately we want ~everyone off of these methods anyway in favor of `localizeConfig`. So, yeah I think we want to clean them up and remove these, but for somewhat different reasons.

Reply:
Actually, we probably can remove `getLocalizeResources`. That's not used by consumers, they just create their own method with that name. Didn't think that one through.

But there are only [7 actual uses](https://github.com/search?q=saved%3AD2L+%40brightspace-ui%2Fcore%2Fhelpers%2FgetLocalizeResources.js&type=code&saved_searches=%5B%7B%22name%22%3A%22D2L%22%2C%22query%22%3A%22org%3ABrightspace+OR+org%3ABrightspaceUI+OR+org%3ABrightspaceUILabs+OR+org%3ABrightspaceHypermediaComponents%22%7D%5D&expanded_query=org%3ABrightspace+org%3ABrightspaceUI+org%3ABrightspaceUILabs+org%3ABrightspaceHypermediaComponents+%40brightspace-ui%2Fcore%2Fhelpers%2FgetLocalizeResources.js) of `getLocalizeOverrideResources` so that seems doable to get cleaned up, or at least audit.